### PR TITLE
Add ansible-lint

### PR DESCRIPTION
**Pull request by adrianmoisey**
Mostly copied from the PEP8 tool.

I haven't tested this on a running copy of lint-review.

### CODE REVIEW

File: tests/tools/test_ansible.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    def test_process_files__one_file_fail(self):
+        self.tool.process_files([self.fixtures[1]])
+        problems = self.problems.all(self.fixtures[1])
+        eq_(11, len(problems))
+
+        fname = self.fixtures[1]
+        expected = Comment(
+            fname, 9, 9,
+            '[EANSIBLE0012] Commands should not change things if nothing needs doing'  # noqa
```
Comment (adrianmoisey) [inline tests/tools/test_ansible.py]: I opted for `noqa` here because I think it's neater then splitting the line.

Comment (markstory) [inline tests/tools/test_ansible.py]: Works for me.

**Comment by markstory** [comment]
The newly added [tests are failing](https://travis-ci.org/markstory/lint-review/builds/175355258).

**Comment by adrianmoisey** [comment]
Doh, it worked locally. I made the PR and left before tests ran. Will figure it out.

**Comment by adrianmoisey** [comment]
I think ansible-lint wasn't sorting the output, so I've added a sort, just so that the result is always the same. Not ideal, but it will work.

**Comment by adrianmoisey** [comment]
I've fixed this upstream in https://github.com/willthames/ansible-lint/pull/219

**Comment by markstory** [comment]
Tagged 0.15.0 which includes these changes.

**Pull request merged**